ipa-core/src/protocol/context/dzkp_validator.rs

```diff
@@ -269,6 +357,17 @@ impl UnverifiedValuesStore {
             }
         }
     }
+
+    /// `get_unverified_field_value` converts a `UnverifiedValuesStore` into an iterator over `UnverifiedFieldValues`
+    /// compatible with DZKPs
+    fn get_unverified_field_values<DF: DZKPBaseField>(
+        &self,
+    ) -> impl Iterator<Item = DF::UnverifiedFieldValues> + '_ {
+        self.vec
+            .iter()
+            .filter_map(|x| x.as_ref())
```

Comment:
I use flatten for now. I think if we want to check the vectors, it would require more consideration (since there are some None values that I don't want to include but they are correct and there might be some None values which indicate that there is an issue)